packages/core/src/util/mime.ts

```diff
@@ -12,10 +12,12 @@ export function isMimeTypeXML(mimeType?: string | undefined | null): boolean {
 
 // define exportable constants for all the different file types
 export const JSON_MIME_TYPE = "application/json";
+export const JSON_MIME_TYPE_2 = "text/json";
 export const JSON_FILE_EXTENSION = ".json";
 export const PDF_MIME_TYPE = "application/pdf";
 export const PDF_FILE_EXTENSION = ".pdf";
 export const XML_MIME_TYPE = "application/xml";
+export const XML_MIME_TYPE_2 = "text/xml";
```

Comment:
Instead of "2" can we rename both to be more clear what one gets based on the name? Like:

```ts
export const XML_APP_MIME_TYPE = "application/xml";
export const XML_TXT_MIME_TYPE = "text/xml";
```